zk/stages/stage_interhashes.go

```diff
@@ -128,15 +128,6 @@ func SpawnZkIntermediateHashesStage(s *stagedsync.StageState, u stagedsync.Unwin
 	eridb := db2.NewEriDb(tx)
 	smt := smt.NewSMT(eridb, false)
 
-	useBatch := shouldIncrement || cfg.zk.SmtRegenerateInMemory
-
-	if useBatch {
-		log.Info(fmt.Sprintf("[%s] SMT using mapmutation", logPrefix))
-		eridb.OpenBatch(quit)
-	} else {
-		log.Info(fmt.Sprintf("[%s] SMT not using mapmutation", logPrefix))
-	}
```

Comment:
like this we don't open a batch for incremental as we previously did.  We should add a link to open the batch inside the `if` below for the `shouldIncrement`